### PR TITLE
Play "reduced motion" marker if `UIAccessibility.isReduceMotionEnabled` is true

### PR DESCRIPTION
**Pull request by calda**
This PR adds support for respecting the system "reduction motion" option.

If a Lottie animation includes a marker named "reduced motion" (or "reducedMotion", "reduced_motion", etc) and `UIAccessibility.isReduceMotionEnabled` is true, that marker will be be played instead of any other animation that was configured.

Here's an example animation that includes a marker named "reduced motion":

| Reduced motion disabled | Reduced motion enabled |
| ---- | ----- |
|  |  |

### CODE REVIEW

File: Sources/Public/Configuration/DecodingStrategy.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Created by Cal Stephens on 7/14/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+/// How animation files should be decoded
+public enum DecodingStrategy: Hashable {
```
Comment (calda) [inline Sources/Public/Configuration/DecodingStrategy.swift:5]: Moved this to its own file

File: Sources/Public/Configuration/LottieConfiguration.swift [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// Created by Cal Stephens on 12/13/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import QuartzCore
+
+/// Global configuration options for Lottie animations
+public struct LottieConfiguration: Hashable {
```
Comment (calda) [inline Sources/Public/Configuration/LottieConfiguration.swift:7]: Moved this to its own file and added the `reducedMotionOption` property

**Comment by calda** [comment]
This change is ready for review. @thedrick, could you review this one?

**Review by thedrick — APPROVED** [review]
this is awesome @calda thanks for adding this feature!

**Pull request merged**